build.gradle

```diff
@@ -70,7 +71,7 @@ subprojects {
         ignoreFailures = true
         sourceSets = [sourceSets.main]
         effort = "max"
-        toolVersion = '2.0.3'
+        toolVersion = '3.0.1'
```

Comment:
yeah you'll have trouble here since our continuous build environment uses Java 6.  We may want to reconsider that.